app/routes.py

```diff
@@ -1 +1,280 @@
-from flask import Blueprint
\ No newline at end of file
+import os
+from datetime import datetime
+
+import requests
+from flask import Blueprint, abort, jsonify, make_response, request
+
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+
+task_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goal_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+##-----------------------------------------------------------------##
+##--------------------------Helper Functions-----------------------##
+##-----------------------------------------------------------------##
+
+def validate_item(cls,id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message": f"{cls.__name__} {id} is invalid"}, 400))
+
+    item = cls.query.get(id)
+    if not item:
+        abort(make_response({"message": f"{cls.__name__} {id} not found"}, 404))
+
+    return item
+
+def validate_new_item(cls,request_body):
```

Comment:
This works well for our use case, since there are only Tasks and Goals, but because you're using an `else` as the Goal case, this functionality isn't very extensible. If you added additional models, and wanted to use this function to validate them, you'd have to modify the else block.